cloud/aws/alb/outputs.tf

```
@@ -0,0 +1,35 @@
+output "alb_4xx_id" {
+  description = "id for detector alb_4xx"
+  value       = signalfx_detector.alb_4xx.*.id
```

```suggestion
  value       = signalfx_detector.alb_4xx[*].id
```
cf https://www.terraform.io/docs/configuration/expressions.html#splat-expressions

---

While we do not use count like for datadog I will remove the list, thanks for the comment.

I updated PR to output the entire resource and not only the id.